### PR TITLE
Bump VF dep to 1.3.x

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Pull in latest version of Vanilla Framework
- Fix up any regressions falling out of VF bump
- Fixed breaking travis build

## QA

- Pull down code
- Run `./run test` and verify there are no errors
- Run `./run serve --watch`
- Check through patterns at http://0.0.0.0:8102/vanilla-brochure-theme/ and make sure none are broken
- Cross ref with Percy and ensure all visual changes are expected
- Ensure travis build passes

## Demo

http://vanilla-brochure-theme-bump-vf.demo.haus/vanilla-brochure-theme/

### CODE REVIEW

**Comment by anthonydillon** [comment]
I am seeing a lot of strip differences and nothing relating to that in the v1.3.0 change notes. Is this something that was introduced by this branch?

**Comment by anthonydillon** [comment]
Missing ticks: https://percy.io/vanilla-framework/vanilla-brochure-theme/builds/271827?snapshot=8358068

**Comment by anthonydillon** [comment]
Percy looks good apart from the missing tick icons.

**Comment by barrymcgee** [comment]
@anthonydillon I see the ticks fine when running locally, do you?

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+  - gem install sass
+  - bundle install
   - npm install
 before_script:
-  - node_modules/gulp/bin/gulp.js build
+  - npm run build
 script:
-  - jekyll build
-  - node_modules/gulp/bin/gulp.js test
+  - npm run test
```
Comment (anthonydillon) [inline .travis.yml]: This works! Nice @nottrobin

**Comment by barrymcgee** [comment]
Demo: http://vanilla-brochure-theme-bump-vf.demo.haus/vanilla-brochure-theme/

**Comment by yaili** [comment]
Why is it that all the strips have a lot more padding? What is the change that has caused it?

**Comment by yaili** [comment]
Other comments from reviewing Percy:
- [ ] There should still be visual separation between the two columns of lists in /vanilla-brochure-theme/examples/patterns/lists/split-list/index.html
- [ ] List ticked has lost the ticks /vanilla-brochure-theme/examples/patterns/lists/ticked-list/index.html
- [ ] Why is strip accent not aubergine?
- [ ] The example of strip image should include an image
- [ ] Vertical divider at small screen needs bottom padding /vanilla-brochure-theme/examples/patterns/divider/index.html

**Comment by barrymcgee** [comment]
- Percy is incorrect, split lists have 1rem of padding on large screens
- Ticks on tick lists are present locally and on demo (currently down), Percy seems to have issues parsing inline SVGs
- Strip accent is no longer aubergine because it inherits $color-brand as default from VF which is grey
- the strip image does include an image, it's just a subtle Suru image

**Comment by barrymcgee** [comment]
@yaili All strip padding now matches the spec as outlined here: https://github.com/ubuntudesign/vanilla-brochure-theme-design/blob/master/Strip/strip.md

**Comment by yaili** [comment]
Just to double check, did you address the last item in my list? It's not ticked.

**Comment by barrymcgee** [comment]
@yaili I see all items ticked? But to confirm, padding under dividers? Yes.

screen shot 2017-07-01 at 10 50 20

**Pull request merged**